core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -56,8 +51,12 @@ public SetupWizard(Jenkins j) throws IOException {
             HudsonPrivateSecurityRealm securityRealm = new HudsonPrivateSecurityRealm(false, false, null);
             j.setSecurityRealm(securityRealm);
             String randomUUID = UUID.randomUUID().toString().replace("-", "").toLowerCase(Locale.ENGLISH);
-            admin = securityRealm.createAccount(SetupWizard.initialSetupAdminUserName, randomUUID);
-            admin.addProperty(new SetupWizard.AuthenticationKey(randomUUID));
+            
+            // create an admin user
+            securityRealm.createAccount(SetupWizard.initialSetupAdminUserName, randomUUID);
+            
+            // JENKINS-33599 - write to a file in the jenkins home directory
```

Comment:
Hmm, I have to disagree with this after thinking about it a bit... someone may install jenkins through apt-get or something similar, which creates a jenkins user, which does not actually allow login, then a normal user who would otherwise have access to JENKINS_HOME would not be able to read the file and perform the installation.